bisect_ppx-ocamlbuild.opam

```diff
@@ -0,0 +1,24 @@
+name: "bisect_ppx-ocamlbuild"
```

Comment:
I also want to be able to pin against some development state and have it "just work" without worrying about version constraints. Unfortunately using `dev` doesn't seem to provide that.